lib/danger/comment_generators/github.md.erb

```diff
@@ -8,10 +8,10 @@
      </tr>
   </thead>
   <tbody>
-    <% table[:content].each do |message| -%>
+    <% table[:content].each do |violation| -%>
     <tr>
       <td>:<%= table[:emoji] %>:</td>
-      <td><%= message %></td>
+      <td data-sticky="<%= violation.sticky %>"><%= violation.message %></td>
```

Comment:
You might want to add this, *only* if it's sticky, also, that would make sure that all previously made comments by Danger would be not-sticky, which I think is desired.